index.html

```diff
@@ -171,4 +171,4 @@ <h3>What's new</h3>
     <script src="analytics/analytics.js"></script>
     <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 </body>
-</html>
+</html
```

Comment:
This change looks unintentional? Lets remove it.

Reply:
oh sorry,I'll change that